octodns/cmds/sync.py

```diff
@@ -22,6 +22,9 @@ def main():
                         help='Acknowledge that significant changes are being '
                         'made and do them')
 
+    parser.add_argument('--lenient', action='store_true', default=False,
+                        help='Do not strictly follow DNS standard.')
```

Comment:
My general worry with this is that `lenient` is also passed to `Zone.add_record` and it allows more than just adding records that don't follow the standards.

https://github.com/github/octodns/blob/7bf4914a41042cd283cf8cecdec88a9e37f2dbb1/octodns/zone.py#L59-L93

This text probably needs to be made more broad. Something to the effect of:

> Allow records that do not conform to DNS standards to be synced as well as records that below in sub-zones

This actually may be a positive thing in terms of allowing the break out of sub-zones (though providers behaviors around this are super wonky/inconsistent.)